package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "demo": "DEBUG=defra.identity:* node demo",
     "postinstall": "cp -n demo/.env.template demo/.env || echo 'Note: Error creating demo/.env. File may already exist'",
-    "test": "PORT=8001 lab -vcL --timeout 20000",
+    "test": "PORT=8001 lab -vdL --timeout 20000",
```

Comment:
Did you want to leave this set to 'dry run'?

Reply:
Good spot - no - have removed